reports/infrastructure-deployments/main_test.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"reflect"
+	"testing"
+)
+
+func Test_getFirstLastDayofMonth(t *testing.T) {
+	type args struct {
+		nthMonth int
+	}
+	tests := []struct {
+		name string
+		args args
+		want date
+	}{
+		// TODO: Add test cases.
```

Comment:
Do we mean to test here?

Reply:
Yes, but they are not yet ready. I have removed until i have a working test